src/OSExtensions/OSExtensions.csproj

```diff
@@ -12,8 +12,12 @@
     <Copyright>Copyright © Microsoft 2010</Copyright>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.Win32.Registry" Version="4.0.0" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.6'">
```

Comment:
I was able to build OSExtensions with net45 rather than net461.   I also removed the dependency on RuntimeInformation (it was causing me failures during testing).  

In short,  you can simply remove your changes for OSExtensions.csproj.  The one that is there now is the one that I built with, and it should work.

Reply:
Alright, that's what I just did